kustomize/resource_kustomization.go

```diff
@@ -177,10 +208,49 @@ func kustomizationResourceRead(d *schema.ResourceData, m interface{}) error {
 	return nil
 }
 
+func kustomizationResourceRead(d *schema.ResourceData, m interface{}) error {
+	kubeconfigPath := d.Get("kubeconfig_path").(string)
+	kubeconfigRaw := d.Get("kubeconfig_raw").(string)
+	kubeContext := d.Get("context").(string)
+
+	client := m.(*Config).Client
+	mapper := m.(*Config).Mapper
+
+	if kubeconfigPath != "" || kubeconfigRaw != "" || kubeContext != "" {
+		k, err := initializeClient(kubeconfigRaw, kubeconfigPath, kubeContext, false, 1)
+
+		if k == nil {
```

Comment:
Should this be `err != nil`?

Reply:
it is technically same, but if k is `nil` here, like 226 will panic, so that seems like a safer check